homeassistant/components/lektrico/coordinator.py

```diff
@@ -0,0 +1,51 @@
+"""Coordinator for the Lektrico Charging Station integration."""
+
+from __future__ import annotations
+
+from datetime import timedelta
+
+from aiohttp import ClientSession
+from lektricowifi import DeviceConnectionError, lektricowifi
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import LOGGER
+
+SCAN_INTERVAL = timedelta(seconds=10)
+
+
+class LektricoDeviceDataUpdateCoordinator(DataUpdateCoordinator[lektricowifi.Info]):
+    """Data update coordinator for Lektrico device."""
+
+    def __init__(
+        self,
+        hass: HomeAssistant,
+        friendly_name: str,
+        host: str,
+        session: ClientSession,
```

Comment:
This session could be created here, this saves another parameter

Reply:
I removed the session parameter and created the session here, thanks.